test_conformance/commonfns/test_step.cpp

```diff
@@ -140,6 +147,16 @@ int test_step_fn(cl_device_id device, cl_context context,
             input_ptr[1][i] = get_random_double(-0x40000000, 0x40000000, d);
         }
     }
+    else if (std::is_same<T, half>::value)
+    {
+        const float fval = 0x40000000;
```

Comment:
I think we need to decrease this value, as the generated data results in infinities most of the time.